LICENSE-binary

```diff
@@ -930,7 +930,7 @@ The license texts of these dependencies can be found in the licenses directory.
         * JAX-RS provider for JSON content type (org.codehaus.jackson:jackson-jaxrs:1.9.13 - http://jackson.codehaus.org)
         * Xml Compatibility extensions for Jackson (org.codehaus.jackson:jackson-xc:1.9.13 - http://jackson.codehaus.org)
         * Javassist (org.javassist:javassist:3.24.1-GA - http://www.javassist.org/)
-        * RocksDB JNI (org.rocksdb:rocksdbjni:5.18.4 - http://rocksdb.org/)
+        * RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org/)
```

Comment:
same issue here I think:
As per the build error, this line needs to be changed to remove the trailing slash:
* RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org/)
should be 
* RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org)

Reply:
@bipinprasad I have update the files as per your suggestion but CI Check-updated-License-files is still failing.

Reply:
In order to find what the license file issue is, you can follow these steps:

1. Click on the checks tab
2. Select the failed job
3. At the bottom you will see content of actual file and expected file.
4. Copy these into two files and two a "diff -w"

I followed these steps for one failed job and see the following result:

```
1,3c1,2
< ******************************Actual is different from expected DEPENDENCY-LICENSES,target/DEPENDENCY-LICENSES******************************
< ****************************** Start of file DEPENDENCY-LICENSES ******************************
< (0) File DEPENDENCY-LICENSES content is:
---
> ****************************** Start of file target/DEPENDENCY-LICENSES ******************************
> (1) File target/DEPENDENCY-LICENSES content is:
537c536
< 	        * RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - http://rocksdb.org)
---
> 	        * RocksDB JNI (org.rocksdb:rocksdbjni:6.27.3 - https://rocksdb.org)
721c720,721
< ****************************** End of file DEPENDENCY-LICENSES ******************************
---
> ****************************** End of file target/DEPENDENCY-LICENSES ******************************
```